Ruangan_901:
  wp1: [3.28, 0.665, 0.0]
  wp2: [3.28, 0.665, 0.0]

Ruangan_903:
  wp1: [3.28, 0.665, 0.0]

Lift_Barat:
  wp1: [3.28, 0.665, 0.0]
  wp2: [3.28, 0.665, 0.0]
